VERSION

```diff
@@ -0,0 +1 @@
+1.6
```

Comment:
What’s this file for?

Reply:
Experiment with releasing; I have the build scripts set up to auto-increment the latter part of the version with each release.

Reply:
Got it. I worry I won’t be the only person to ask!

I wonder if we’d be better to have something like “scripts/BuildConfig.sh” which had

```
# a comment
VERSIONBASE=1.6
```

Type thing so we don’t end up with one file per feature and we can explain it a bit?

I guess it all depends on how you are reading VERSION but you get the idea!

Reply:
I don't much care. Just need some way to auto-increment a version number. Are there any standard ways of doing this in the c++ world? I'm spoiled by [Maven](maven.apache.org/maven-release/maven-release-plugin/)

Reply:
Not standard ones no. Where do you use the file?

Reply:
Ok cool so you do something like cat it with backticks?

Thinking we will have other info we want to pass to the auto build scripts (I’m thinking things like signature files and stuff in the future) would it be possible to have a file which just defines surge version base (using = assigned to a name) and then include that so we could set other things later and include a comment? I worry the single file called version with a single number 1.6 could be a little bit confusing to a future person figuring this out.